public/javascripts/inquiry.js

```diff
@@ -1,7 +1,22 @@
 $(function () {
+  const noteStore = lichess.storage.make('inquiry-note');
+  const noteTextArea = $('#inquiry .notes').find('textarea')[0];
+
   $('#inquiry .notes').on('mouseenter', function () {
-    $(this).find('textarea')[0].focus();
+    noteTextArea.focus();
+    noteTextArea.value = noteStore.get();
+  });
+
+  $('#inquiry .notes').on('keydown', function () {
+    setTimeout(function () {
+      noteStore.set(noteTextArea.value);
+    }, 5);
   });
+
+  $('#inquiry .notes .button').on('click', function () {
+    noteStore.remove();
+  });
```

Comment:
what if I use Tab then Enter to save the note?

Reply:
maybe bind on form submit event instead

Reply:
Pressing enter doesn’t save the note

Reply:
I’ll look to change on form submit